themes/default/content/what-is/what-are-kubernetes-secrets.md

```diff
@@ -0,0 +1,275 @@
+---
+title: What are Kubernetes Secrets?
+meta_desc: |
+     Learn about Kubernetes Secrets and how to manage sensitive information securely in your Kubernetes clusters.
+
+type: what-is
+page_title: "What are Kubernetes Secrets"
+
+customer_logos:
+  title: Leading engineering organizations are building with Pulumi
+  logos:
+    - items:
+      - snowflake
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+Kubernetes, or K8S, is an open-source container orchestration platform designed to automate the deployment, scaling, and management of containerized applications.
```

Comment:
not a big deal, but I typically see K8S, with the "s" being lowercase in most other places, but not sure if there is an "official" formatting of this.